rotkehlchen/inquirer.py

```diff
@@ -479,6 +481,9 @@ def _try_oracle_price_query(
         - RecursionError if `coming_from_latest_price` is True. Used in the ManualCurrentOracle
         """
         price, used_main_currency, is_error = ZERO_PRICE, False, True
+
+        if from_asset == A_POLYGON_POS_MATIC and ts_now() > POLYGON_POS_POL_HARDFORK:  # after hardfork, we use different oracles  # noqa: E501
+            from_asset = AssetWithOracles('eip155:1/erc20:0x455e53CBB86018Ac2B8092FdCd39d8444aFFC3F6')  # POL token  # noqa: E501
```

Comment:
calling the constructor of `AssetWithOracles` is the same as calling `EvmToken` right? Maybe is clearer to use `EvmToken`

Reply:
I used it for efficiency because initializing EvmToken will be more expensive as it holds a lot of other fields that are not needed here